content/en/common/kubernetes-versions.md

```diff
@@ -5,6 +5,6 @@ _build:
 
 | Installation Tool | KubeSphere version | Supported Kubernetes versions                                |
 | ----------------- | ------------------ | ------------------------------------------------------------ |
-| KubeKey           | v3.1.0             | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
-| ks-installer      | v3.1.0             | v1.17.x, v1.18.x, v1.19.x, v1.20.x                           |
+| KubeKey           | v3.1.1             | v1.17.0, v1.17.4, v1.17.5, v1.17.6, v1.17.7, v1.17.8, v1.17.9, v1.18.3, v1.18.5, v1.18.6, v1.18.8, v1.19.0, v1.19.8, v1.19.9, v1.20.4 |
```

Comment:
Please add `v1.20.6` to `Supported Kubernetes versions`.
<img width="810" alt="Xnip2021-07-21_15-18-43" src="https://user-images.githubusercontent.com/75110798/126447919-95db49e4-c100-433d-805c-9bb94c28df06.png">

Reply:
Added.